rpg/urls.py

```diff
@@ -27,4 +27,7 @@
     path(
         r"forge/<int:student_pk>/", views.DiamondUpdate.as_view(), name="diamond-update"
     ),
+    path(
+        r"stats/<int:student_pk>/", views.stats, name="heading-charisma"
+    ),
```

Comment:
That's what it was originally.

I think this approach is fine too, but in that case adding a new URL path named "charisma-heading" is completely unnecessary because it is identical to the existing "stats" view. Just delete all the changes here and use "stats" in the template.